### DISCUSSION TITLE
Frustrating html snippet on react files

### DISCUSSION THREAD

**Discussion by Coder-Vasen** [NvChad/NvChad#2899] [Q&A] [answered]
I am new to neovim environment, got attracted by nvchad simplicity and elegance. I had setup plugins for web development. In react file I am getting html snippet as first auto completion suggestion
![Screenshot-8](https://github.com/NvChad/NvChad/assets/69237691/c43c7ac1-f847-48dc-8799-b339f9c42038)

**Expected behaviour**

-  No html snippet on javascript code area
-  typescript lsp should have priority in markup area when curly braces `{}` are used

**Comment by siduck** [comment, marked as answer]
that isnt related to nvchad .     Nvchad doesnt handle any language specific stuffs except for LUA

ask on r/neovim :D
